std/archive/README.md

```diff
@@ -3,7 +3,7 @@
 ## Tar
 
 ```ts
-import { Tar } from "https://deno.land/std/archive/tar.ts";
+import { Tar } from "https://deno.land/std@0.70.0/archive/tar.ts";
```

Comment:
@tokiedokie this is nice, but the problem is we'd have to update it with each release and we're already at `0.71.0`. I chatted with @lucacasonato and we could handle this more gracefully by using `$STD_VERSION` tag in `std`'s READMEs (similarly to files in `docs/` directory).

Reply:
done